js/build.js

```diff
@@ -1,5 +1,26 @@
 Handlebars.registerHelper('formatMessage', function(text) {
-  text = Handlebars.Utils.escapeExpression(text).replace(/(\r\n|\n|\r)/gm, '<br>');
+  var breakRegExp = /(\r\n|\n|\r)/gm,
+      emailRegExp = /(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))/gm,
+      numberRegExp = /[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,8}/gm,
+      urlRegExp = /(?:^|[^@\.\w-])([a-z0-9]+:\/\/)?(\w(?!ailto:)\w+:\w+@)?([\w.-]+\.[a-z]{2,4})(:[0-9]+)?(\/.*)?(?=$|[^@\.\w-])/ig;
+
+  /* capture email addresses and turn into mailto links */
+  text = text.replace(emailRegExp, '<a href="mailto:$&">$&</a>');
+
+  /* capture phone numbers and turn into tel links */
+  text = text.replace(numberRegExp, '<a href="tel:$&">$&</a>');
+
+  /* capture URLs and turn into links */
+  /* @TODO: Remove line breaks, spaces from results
```

Comment:
Do we still need to do this? If not, remove the `@TODO`. If so, what does this mean?